apps/chat/models.py

```diff
@@ -209,6 +209,19 @@ def add_version_tag(self, version_number: int, is_a_version: bool):
             tag = f"{tag}-unreleased"
         self.add_system_tag(tag=tag, tag_category=TagCategories.EXPERIMENT_VERSION)
 
+    def add_rating(self, tag: str):
+        tag, _ = Tag.objects.get_or_create(
+            name=tag,
+            team=self.chat.team,
+            is_system_tag=False,
+            category=TagCategories.RESPONSE_RATING,
+        )
+        self.add_tag(tag, team=self.chat.team, added_by=None)
```

Comment:
I like having the participant being anonymous, but then having the added_by=None feels not intuitive along with some of the filtering-- I'm curious if rather than a category if you considered going the route of a `is_system_tag` that would also open the possibly of more participant guided tags in the future (which I _think_ was discussed vaguely but perhaps not)-- either approach works just bringing up the idea

Reply:
Good question. I did consider the system tag route, but I think we should keep it reserved for case where the server created and added the tag, with no human intervention. I agree that we can and should improve the meaning of system tag vs user tag vs participant tag. I think let's keep things as-is for now until we have a clearer solution to this. A mini spec might do.

There are two fundamental questions to answer:

1. Who created this tag?
2. Who added this tag to this message?

For the most part question 2 is the most interesting one for users I'd imagine. As for determining this, we could do it either at the application layer or DB layer. In the latter case, we can have a flag or something on the `CustomTaggedItem` model to indicate which entity added this to the message i.e. user/system/participant.

Reply:
@coderabbitai please create a ticket for this.